api/v1alpha1/gateway_types.go

```diff
@@ -214,6 +214,10 @@ type Listener struct {
 	// Support: Core
 	//
 	// +required
+	// +kubebuilder:validation:Minimum=1
+	// +kubebuilder:validation:Maximum=65536
+	// +kubebuilder:validation:ExclusiveMinimum=false
+	// +kubebuilder:validation:ExclusiveMaximum=true
```

Comment:
Could we make these either both true or both false?

Reply:
Will do.